### PR TITLE
Add -y for apt-get build-dep for the debian installation

### PR DESCRIPTION
**Pull request by matrixise**

### CODE REVIEW

File: build_cpython_linux.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Get tools and dependencies
         $ sudo dnf install dnf-plugins-core
         $ sudo dnf builddep python3
 
-  * Debian: ``sudo apt-get build-dep python3``
+  * Debian: ``sudo apt-get build-dep -y python3``
```
Comment (tao12345666333) [inline build_cpython_linux.rst:35]: If the -y option is added here, do other parts also need to be added? e.g. `yum`/`dnf` etc.

**Review by tao12345666333 — COMMENTED** [review]
Thanks, add one comment.

File: build_cpython_linux.rst (hunk 1 above)
Comment (matrixise) [reply build_cpython_linux.rst:35]: The scope of this PR is just for the Debian system

Comment (tao12345666333) [reply build_cpython_linux.rst:35]: > The scope of this PR is just for the Debian system

OK. And should we open another PR for other system? 

After #16 has been merged, we need resolve conflicting files for this PR. :smile_cat:

Comment (matrixise) [reply build_cpython_linux.rst:35]: fixed, thank you. now, if you want to contribute with a PR for Fedora/Redhat, let's go ;-)

Comment (tao12345666333) [reply build_cpython_linux.rst:35]: OK, I will do it. Thanks.

**Pull request closed without merging**